developer/src/tike/project/Keyman.Developer.System.Project.ProjectFile.pas

```diff
@@ -1154,64 +1147,6 @@ procedure TProjectFileStates.SetItem(Index, Value: WideString);
   end;
 end;
 
-const
-  MAX_MESSAGES = 100;
-
-var
-  MessageCount: Integer = 0;
-
-procedure ProjectCompilerMessageClear;
-begin
-  MessageCount := 0;
-end;
-
-function ProjectCompilerMessage(line: Integer; msgcode: LongWord; text: PAnsiChar): Integer; stdcall;  // I3310   // I4694
-begin
-  Result := ProjectCompilerMessageW(line, msgcode, String_AtoU(text));
-end;
-
-function ProjectCompilerMessageW(line: Integer; msgcode: LongWord; const text: string): Integer;  // I3310   // I4694
-const // from compile.pas
```

Comment:
These functions no longer used because they were part of the old compiler.pas interface.